src/lib/request-security.ts

```
@@ -379,19 +423,21 @@ export function blockSuspiciousPathsMiddleware(req: Request, res: Response, next
+				kind: 'blocked_path',
+				action: 'blocked',
+				statusCode: 403,
+				matchedRule,
+			}))
+			.catch((error) => {
+				logger.error('Failed to persist blocked-path security activity', {
+					error: error instanceof Error ? error.message : String(error),
+					path: req.path,
+					ipHash: baseFingerprint.ipHash,
+				});
 			});
```

ESLint

[error] Expected 'undefined' and instead saw 'void'.

(no-void)